Gruntfile.js

```diff
@@ -78,22 +73,37 @@ module.exports = function (grunt) {
 			},
 
 			coverageUnit: {
-				command: 'node test/unit/index.js',
+				command: 'TEST_TYPE=unit && node test/unit/index.js',
 				maxBuffer: maxBufferSize
 			},
 
 			coverageUnitSlow: {
-				command: 'node_modules/.bin/istanbul cover --dir test/.coverage-unit ./node_modules/.bin/_mocha test/unit/index.slow.js',
+				command: 'TEST_TYPE=unit && node_modules/.bin/istanbul cover --dir test/.coverage-unit ./node_modules/.bin/_mocha test/unit/index.slow.js',
 				maxBuffer: maxBufferSize
 			},
 
 			testFunctionalWs: {
-				command: './node_modules/.bin/_mocha test/functional/ws/index.js',
+				command: 'export NODE_ENV=test TEST_TYPE=func && node_modules/.bin/istanbul cover --dir test/.coverage-unit --include-pid ./node_modules/.bin/_mocha test/functional/ws/index.js -- --grep @slow --invert',
```

Comment:
Are these blocks for coverage or for running tests? Because I don't understand the order here. `coverageUnitSlow`, `testFunctionalWs`, `testFunctionalHttpGet` call istanbul, while `testFunctionalWsExtensive`, `testFunctionalHttpGetExtensive` run _mocha

Reply:
I plan to review Gruntfile and coverage in another PR. I don't want this one to keep growing.